src/Marten/Linq/Filters/FullTextWhereFragment.cs

```diff
@@ -27,6 +27,12 @@ public FullTextWhereFragment(DocumentMapping mapping, FullTextSearchFunction sea
             _searchTerm = searchTerm;
         }
 
+        private string SanitizeSearchTerm(string searchTerm)
+        {
+            // edge case that will cause a sql exception with a single quote we need to handle also
+            return searchTerm == "'" ? "" : searchTerm?.Replace("'", "''");
```

Comment:
If you find a single quote in text, it will need to replaced by double single quotes `''` but the above code looks to be incorrect i.e. if the search term contains single quote, it is returning empty string and if not then it is trying to do a replace. shouldn't it be just `return searchTerm?.Replace("'", "''");`? Please recheck this.

Could you please add a unit test specifically with search term containing semicolon?

Reply:
> https://github.com/npgsql/efcore.pg/pull/1888 - this is the EFCore issue for reference.
@phinett Quick follow on note, I re-read the issue in the efcore.pg repo and observing that only the config doesn't need to be parametrized and search term can be parametrized. So I would suggest to keep the config as string and leave the search term as parameter i.e. `:argSearchTerm`. This way, we don't need to do any sanitization on the search term.

Reply:
I have added the above suggested fix.